lib/engine/game/g_18_eus/game.rb

```diff
@@ -290,20 +318,29 @@ def stock_round
 
         def operating_round(round_num)
           Engine::Round::Operating.new(self, [
-            Engine::Step::Bankrupt,
+            G18EUS::Step::Bankrupt,
             Engine::Step::Exchange,
+            Engine::Step::DiscardTrain,
             G18EUS::Step::SpecialTrack,
-            Engine::Step::BuyCompany,
```

Comment:
I put a "skip" in for this step - will need to change that to acquire.  I can do that after your gets merged in.